openassessment/xblock/rubric_reuse_mixin.py

```diff
@@ -0,0 +1,82 @@
+""" Mixin for functionality around the reuse of rubrics between ORAs """
+from opaque_keys import InvalidKeyError
+from opaque_keys.edx.keys import UsageKey
+from xblock.core import XBlock
+
+
+class RubricReuseMixin:
+
+    def _get_course_ora_blocks(self):
+        """
+        Get all ORA blocks from the course.
+        Both draft and published versions are queried, with preference to draft versions
+        """
+        from xmodule.modulestore.django import modulestore
+
+        return modulestore().get_items(
```

Comment:
Any sense of having to bake in more course/role isolation? e.g. could non-privelaged users or admins from other courses abuse this?

I'm thinking no, but want to make sure.